File: JSBASIC/137_diamond.js
// 137	137_diamond.js	Print a full diamond star pattern using loops.

let n = 5;
console.log("Diamond pattern:");
for (let i = 1; i <= n; i++) {
    console.log(" ".repeat(n - i) + "*".repeat(2 * i - 1));
}
for (let i = n - 1; i >= 1; i--) {
    console.log(" ".repeat(n - i) + "*".repeat(2 * i - 1));
}

File: JavaBasic/137_diamond.java
// 137	137_diamond.java	Print a full diamond star pattern using loops.
public class _137_diamond {
    public static void main(String[] args) {
        int n = 5;
        for (int i = 1; i <= n; i++) {
            for (int s = 1; s <= n-i; s++) System.out.print(" ");
            for (int j = 1; j <= 2*i-1; j++) System.out.print("*");
            System.out.println();
        }
        for (int i = n-1; i >= 1; i--) {
            for (int s = 1; s <= n-i; s++) System.out.print(" ");
            for (int j = 1; j <= 2*i-1; j++) System.out.print("*");
            System.out.println();
        }
    }
}
